gborah-OKTA-402993 add PROFILE_ENROLLMENT and ACCESS_POLICY in Policy Object table (#2213)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/reference/api/policy/index.md

| Parameter     | Description                                                        | Data Type                                      | Required   | Default                |
| :------------ | :----------------------------------------------------------------- | :--------------------------------------------- | :--------- | :--------------------- |
| id            | Identifier of the Rule                                             | String                                         | No         | Assigned               |

> | type          | Rule type. Valid values: `SIGN_ON` or `PASSWORD` or `MFA_ENROLL` or `IDP_DISCOVERY`. <ApiLifecycle access="ie" /><br> <br>**Note:** Following policy types are only available as a part of the Okta Identity Engine. Please [contact support](mailto:dev-inquiries@okta.com) for further information: <br>`ACCESS_POLICY` or `PROFILE_ENROLLMENT`  | String (Enum)                                  | Yes        |                        |

Reviewer: ```suggestion
| type          | Rule type. Valid values: `SIGN_ON`, `PASSWORD`, `MFA_ENROLL`, `IDP_DISCOVERY`. <ApiLifecycle access="ie" /><br> <br>**Note:** The following policy types are available only with Identity Engine: <br>`ACCESS_POLICY`, `PROFILE_ENROLLMENT` [Contact support](mailto:dev-inquiries@okta.com) for more information.  | String (Enum)                                  | Yes        |                        |
```
